lib/v2/shmeea/index.js

```diff
@@ -1,44 +1,49 @@
 const got = require('@/utils/got');
 const cheerio = require('cheerio');
+const timezone = require('@/utils/timezone');
+const { parseDate } = require('@/utils/parse-date');
 
 module.exports = async (ctx) => {
-    const baseURL = 'http://www.shmeea.edu.cn';
-    const rootUrl = baseURL + '/page/08000/index.html';
-    const response = await got({
-        method: 'get',
-        url: rootUrl,
-    });
+    const id = ctx.params.id;
```

Comment:
It would be better to make `id` an optional parameter with default value of `08000` so that those subscribing to `/shmeea` won't have to update their feed url.